server/service/users.go

```diff
@@ -399,6 +435,8 @@ func getInfoAboutSessionsForUserEndpoint(ctx context.Context, request interface{
 }
 
 func (svc *Service) GetInfoAboutSessionsForUser(ctx context.Context, id uint) ([]*fleet.Session, error) {
+	// TODO(mna): should that be ActionRead? It does update the session's access
+	// time, but the same-ish access in GetInfoAboutSession is an ActionRead.
 	if err := svc.authz.Authorize(ctx, &fleet.Session{UserID: id}, fleet.ActionWrite); err != nil {
```

Comment:
^ see comment

Reply:
Hm, yeah, sounds like it should be read. @zwass do you know if there are any other reasons for this to be Write?